src/software/jetson_nano/primitive_executor.h

```diff
@@ -90,7 +90,7 @@ class PrimitiveExecutor
     RobotId robot_id_;
 
     // Estimated delay between a vision frame to AI processing to robot executing
-    static constexpr double VISION_TO_ROBOT_DELAY_S = 0.03;
+    static constexpr double VISION_TO_ROBOT_DELAY_S = 0.06;
```

Comment:
This change breaks dribbling in sim. We should pass it in as an argument to the `PrimitiveExecutor` with different values depending on if it's being initialized in `ErForceSimulator` or `Thunderloop`. Those values can probably be stored as constants in their respective `.h` files.